client/lib/url/test/common/url-type.skip.js

```diff
@@ -53,8 +45,6 @@ describe( 'determineUrlType', () => {
 		expect( determineUrlType( 0 ) ).toBe( 'INVALID' );
 		expect( determineUrlType( '///' ) ).toBe( 'INVALID' );
 		// From https://url.spec.whatwg.org/#urls
-		expect( determineUrlType( 'https://ex ample.org/' ) ).toBe( 'INVALID' );
 		expect( determineUrlType( 'https://example.com:demo' ) ).toBe( 'INVALID' );
-		expect( determineUrlType( 'http://[www.example.com]/' ) ).toBe( 'INVALID' );
```

Comment:
Were these bad test cases?

Reply:
They failed in the polyfill, and I felt they don't add much to the tests; they'd be testing the polyfill/native implementation themselves more than `determineUrlType`.